### ISSUE TITLE
Isolated Scope and ng-include

### ISSUE THREAD

**Issue by LennyLip** [ghiscoding/angular-validation#62]
i've read this https://github.com/ghiscoding/angular-validation/wiki/Isolated-Scope and don't understand , how to using angular-validation with ng-include.

Say, we have index.html

    <form novalidate name="nameForm">
    <input name="field1" validation="required">
    <div ng-inlude="template.html">
    </form>

template.html:

    <input name="field2" validation="required">

field1  validate is ok, but how to validate field2?

**Comment by ghiscoding** [comment]
The `isolatedScope` property was added to force the use of a certain scope, I added it when using Modal Window, since that uses an isolated scope I came up with this new property. So just use it when you know you get an isolated scope and when you know it then just supply it to your definition and then you would need to use that same scope for checking the form.

I almost never use the `ngInclude`, I use instead the `ngView` so I can't help you with that.
